2do_fe/src/components/TaskDetail/TaskDetail.js

```diff
@@ -1,22 +1,23 @@
 import React from "react";
-//import { useDispatch } from "react-redux";
+import { useDispatch } from "react-redux";
 import ProgressIcon from "./ProgressIcon.svg";
 import FinishIcon from "./FinishIcon.svg";
 import StageButton from "../Done/StageButton.js";
 import { openEditDesc } from "../PopUps/EditTaskPopUp";
 import RemoveCircleIcon from "@mui/icons-material/RemoveCircle";
-//import { openTaskDesc } from "../AddTask/AddTaskPopUp.js";
+// import { openTaskDesc } from "../AddTask/AddTaskPopUp.js";
```

Comment:
remove import { openTaskDesc } from "../AddTask/AddTaskPopUp.js ?

Reply:
Got it cleaning it up!